node/src/neighborhood/gossip_acceptor.rs

```diff
@@ -3297,10 +3642,8 @@ mod tests {
         let mut expected_dest_db = src_db.clone();
         expected_dest_db.add_arbitrary_half_neighbor(dest_node.public_key(), src_node.public_key());
```

Comment:
Oh...I suppose now that this might have something to do with the goal of the test. As if we wanted to prove that we couldn't make the src_node a full_neighbor because the degree is too high? 

However, I don't understand why the further `Src` Node had to be linked by the `Dest`. Who wanted them to try being close neighbors? Shouldn't this be a job of Debut, Introduction or Pass? I'll be happy for some explanation.